property/index.js

```diff
@@ -166,3 +161,62 @@ module.exports = yeoman.generators.Base.extend({
 
   saveProject: actions.saveProject
 });
+
+function coerceDefault(def, value) {
+  var isSupported = typeChoices.indexOf(def.type) !== -1;
+  debug('property type "%s" >> supported: %s', def.type, isSupported);
+
+  if (isSupported) {
+    switch ((def.type || '').toLowerCase()) {
+      case 'string':
+        def.default = value;
+        break;
+      case 'number':
+        def.default = Number(value);
+        break;
+      case 'boolean':
+        if (['true', '1', 't'].indexOf(value) !== -1 ){
+          def.default = true;
+        } else {
+          def.default = false;
+        }
+        break;
```

Comment:
For this case, can you just do `Boolean(value)`?  Or are you saying that default `true` has to explicitly be `true`, `1`, or `t`? 

As a JavaScript developer, I would expect any truthy value to coerce to `true`.

@bajtos might have a good comment though as he has just dealt (or is still dealing) with a bunch of LoopBack coercion issues.